src/main/MQ2ImGuiConsole.cpp

```diff
@@ -1554,6 +1587,39 @@ static void MakeColorGradient(float frequency1, float frequency2, float frequenc
 	WriteChatColor(szBuffer);
 }
 
+void FontSizePicker()
+{
+	for (int i = 0; i < s_fontSizeInfo.size(); ++i)
+	{
+		if (s_fontSizeInfo[i].value == s_fontSize)
+		{
+			s_currentFontIndex = i;
+			break;
+		}
+	}
```

Comment:
this is why that vector should be a map

Reply:
actually, looking at this a little closer, this is completely unnecessary. You _only_ need `s_fontSize`